app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -250,7 +250,8 @@ def client
           :ovirt_password   => password,
           :ovirt_url        => url,
           :ovirt_datacenter => uuid,
-          :ovirt_ca_cert_store => ca_cert_store(public_key)
+          :ovirt_ca_cert_store => ca_cert_store(public_key),
+          :ovirt_filtered_api => filtered_api
```

Comment:
this was introduced in fog 1.28, we are already on [1.37](49569b06bd38e508078f9e1257467da13596ef00), rbovirt seems to support filtered API since [v0.0.24](https://github.com/abenari/rbovirt/commit/494028e14948d89ab331e79d6e64fdfeec5bae4c) and were using [0.0.32+](https://github.com/abenari/rbovirt/commit/494028e14948d89ab331e79d6e64fdfeec5bae4c) not sure about when they added it in oVirt but I guess if users oVirt does not support it, they can simply keep it disabled (I think it would be very old version)